macos-setup.md

```diff
@@ -273,6 +273,37 @@ and then change directory into it:
 cd ~/devacademy
 ```
 
+## 5.1 Generate an SSH key pair
+
+There's a good chance you have one of these, you can see a list of your public keys like this:
+
+```sh
+ls ~/.ssh/*.pub
```

Comment:
`ls -al ~/.ssh/*.pub` ?

Reply:
the original line works for me, why those flags?

Reply:
Coz I said so.

Haha, nah - I just found a guide online that used -al.  I'm not fussed whether you do it - just added in case it helps some students.